src/rpc/mempool.cpp

```diff
@@ -44,11 +46,21 @@ static RPCHelpMan sendrawtransaction()
 {
     return RPCHelpMan{
         "sendrawtransaction",
-        "Submit a raw transaction (serialized, hex-encoded) to local node and network.\n"
-        "\nThe transaction will be sent unconditionally to all peers, so using sendrawtransaction\n"
-        "for manual rebroadcast may degrade privacy by leaking the transaction's origin, as\n"
-        "nodes will normally not rebroadcast non-wallet transactions already in their mempool.\n"
+        "Submit a raw transaction (serialized, hex-encoded) to the network.\n"
+
+        "\nIf -privatebroadcast is disabled, then the transaction will be put into the\n"
+        "local mempool of the node and will be sent unconditionally to all currently\n"
+        "connected peers, so using sendrawtransaction for manual rebroadcast will degrade\n"
+        "privacy by leaking the transaction's origin, as nodes will normally not\n"
+        "rebroadcast non-wallet transactions already in their mempool.\n"
+
+        "\nIf -privatebroadcast is enabled, then the transaction will be sent only via\n"
```

Comment:
we likely need a release notes for this, many people would want to know about it